docs/README.md

```diff
@@ -31,38 +31,29 @@ For support and questions, visit [the NeoForged Discord server][Discord].
 
 [See the "Getting Started" section in the NeoForged Documentation][Getting-Started].
 
-# Contribute to Forge
+### Contribute to NeoForge
```

Comment:
The level of this header should be equal to the level of the `Creating Mods` header. Either this header should be H1 (`#`), or this and the other headers above should be H2 (`##`).

Reply:
I have changed this header and the others to H2 since I feel like it would be a better choice.